main.sh

```diff
@@ -287,12 +255,11 @@ for test in "$@"; do
             killall_proc $test
             ;;
         MDTEST)
-            #start_server
-            #start_agent
-            #create_pool
-            #run_mdtest
-            #killall_proc $test
-	    echo "WIP to enable MDTEST on daos master. Please use older commit for running MDTEST on 1.0.0"
+            start_server
+            start_agent
+            create_pool
+            run_mdtest
```

Comment:
Same comment as above

Reply:
I will leave that to you in future patch since it doesn't affect the mdtest results in the use case here.